src/extension.ts

```diff
@@ -18,39 +18,56 @@ import {
 } from "./commands";
 import { ProsProjectEditorProvider } from "./views/editor";
 import { Analytics } from "./ga";
-import { install, configurePaths, uninstall, updateCLI } from "./one-click/install";
+import {
+  install,
+  configurePaths,
+  uninstall,
+  updateCLI,
+} from "./one-click/install";
 import { TextDecoder, TextEncoder } from "util";
 let analytics: Analytics;
 
-export var system : string;
+export var system: string;
 export const output = vscode.window.createOutputChannel("PROS Output");
 
-/// Get a reference to the "PROS Terminal" VSCode terminal used for running 
+/// Get a reference to the "PROS Terminal" VSCode terminal used for running
 /// commands.
-export const getProsTerminal = async (context: vscode.ExtensionContext): Promise<vscode.Terminal> => {
-  const prosTerminals = vscode.window.terminals.filter(t => t.name === "PROS Terminal");
+export const getProsTerminal = async (
+  context: vscode.ExtensionContext
+): Promise<vscode.Terminal> => {
+  const prosTerminals = vscode.window.terminals.filter(
+    (t) => t.name === "PROS Terminal"
+  );
   if (prosTerminals.length > 1) {
     // Clean up duplicate terminals
-    prosTerminals.slice(1).forEach(t => t.dispose());
+    prosTerminals.slice(1).forEach((t) => t.dispose());
   }
 
   // Create a new PROS Terminal if one doesn't exist
-  if (prosTerminals.length) {
+  if (prosTerminals.length && process.env["PATH"]?.includes("pros-cli")) {
```

Comment:
we don't re-use an existing terminal if we couldn't set the PATH for it. When testing I would get stuck with a terminal without the PATH set correctly if the installation failed.

Reply:
Good suggestion, I've updated my code to do that!